src/startupscriptgenerator/node/scriptgenerator.go

```diff
@@ -135,6 +135,14 @@ func (gen *NodeStartupScriptGenerator) GenerateEntrypointScript() string {
 		startupCommand = common.ExtendPathForCommand(startupCommand, gen.SourcePath)
 	}
 
+	logger.LogInformation("Checking configuration to setup appinsight auto collection if needed")
```

Comment:
There's no configuration check actually being done here - the only thing that happens is that the Bash script is **written**. It'll be executed at a later point in time